### PR TITLE
chore(exporter/app): migrate to Zeebe 0.15.0

### LINKED ISSUE #54: Migrate to Zeebe 0.15.0
With Zeebe 0.15.0, the lifecycle of the events has changed. We need to adopt the changed in order to show the correct state of the instance.

### PR DESCRIPTION
**Pull request by saig0**
* migrate to new client api
* migrate to new event lifecycle
* rename database table and columns

closes #54

### CODE REVIEW

File: app/src/main/java/io/zeebe/monitor/rest/WorkflowInstanceResource.java [hunk 1]
```diff
@@ -69,13 +63,7 @@ public void updateRetries(@PathVariable("key") long key) throws Exception {
...
 
               if (jobKey > 0) {
-                connections
-                    .getClient()
-                    .jobClient()
-                    .newUpdateRetriesCommand(jobKey)
-                    .retries(2)
-                    .send()
-                    .join();
+                connections.getClient().newUpdateRetriesCommand(jobKey).retries(2).send().join();
```
[the remark this answers is not in the document]
Comment (saig0) [reply app/src/main/java/io/zeebe/monitor/rest/WorkflowInstanceResource.java]: Good catch 😂 Some leftovers...

[the remark this answers is not in the document]
Comment (Zelldon) [reply app/src/main/java/io/zeebe/monitor/rest/WorkflowInstanceResource.java]: Should I change it to 1? Or what is the fix for this?

**Comment by saig0** [comment]
This method is not used anymore. We can delete it 😉

File: app/src/main/java/io/zeebe/monitor/rest/WorkflowInstanceResource.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (saig0) [reply app/src/main/java/io/zeebe/monitor/rest/WorkflowInstanceResource.java]: This method is not used anymore. I'll remove with the next PR: https://github.com/zeebe-io/zeebe-simple-monitor/commit/250aa063ba6239a19407266e2afc202ab89e6975 ;)

**Pull request merged**